src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -55,15 +55,15 @@ import styles from './Browser.scss';
 import 'static/browser/browser.js';
 
 type StateProps = {
-  activeGenomeId: string | null;
```

Comment:
But it can be null! If storage-service does not find active genome id in localStorage, it will return null:

https://github.com/Ensembl/ensembl-client/blob/9ac0015a723c8148fb3c5c5eaf70ab158d346442/src/ensembl/src/services/storage-service.ts#L55

The typing here is incorrect:
https://github.com/Ensembl/ensembl-client/blob/e963f12cd513415b7ceb7b1bdb1f9fbc4f428eae/src/ensembl/src/content/app/browser/browser-storage-service.ts#L29

That’s why I added the nulls during the last refactor.